Analysis/Tasks/Utils/include/AnalysisTasksUtils/UtilsDebugLcK0Sp.h

```diff
@@ -10,27 +10,54 @@
+  assert(listLabelsK0SPos.size() == listLabelsK0SPos.size());
+  for (auto i = 0; i < listLabelsK0SPos.size(); ++i) {
```

Comment:
Consider using `std::any_of` instead of a for loop.

Reply:
Done